server/Entities/Models/News.cs

```diff
@@ -0,0 +1,22 @@
+using Entities.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebAPI.Models;
+
+namespace Entities.Models
+{
+    public class News
+    {
+        public int NewsId { get; set; }
+        public string NewsTitle { get; set; }
+        public string NewsContent { get; set; }
```

Comment:
it is entity News, it is obvious that properties belongs News, so just Id, Title and Content

Reply:
done